en/topics/databases/SQL/README.md

```diff
@@ -4,11 +4,11 @@
 
 ## What is SQL? 
 
-What is a database and an RDMS? This intro article covers the basics and some flavors of SQL 
+What is a database and an RDBMS? This intro article covers the basics and some flavors of SQL 
 
 - [What is a Relational Database Management System?](https://www.codecademy.com/articles/what-is-rdbms-sql)
 
-> SQL (Structured Query Language) is a domain-specific language used in programming and designed for managing data held in a relational database management system (RDBMS), or for stream processing in a relational data stream management system (RDSMS). It is particularly useful in handling structured data, i.e. data incorporating relations among entities and variables. [Wikipedia](https://en.wikipedia.org/wiki/SQL)
+> SQL (Structured Query Language) is a domain-specific language used in programming and designed for managing data held in a relational database management system (RDBMS), or for stream processing in a relational data stream management system (RDBMS). It is particularly useful in handling structured data, i.e. data incorporating relations among entities and variables. [Wikipedia](https://en.wikipedia.org/wiki/SQL)
```

Comment:
This was already correct as it uses "stream" and I copied this verbatim from Wikipedia.

Reply:
Sorry, I thought it was a distraction error